packages/checkouts/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# @thirdweb-dev/wallets
```

Comment:
not sure why this says `wallets` in it - this file should be auto generated via `changesets` so probably safe to delete